package.json

```diff
@@ -13,7 +13,7 @@
   },
   "dependencies": {
     "@faker-js/faker": "7.6.0",
-    "@iqss/dataverse-client-javascript": "2.0.0-pr99.c36f1db",
+    "@iqss/dataverse-client-javascript": "2.0.0-pr105.779a3f4",
```

Comment:
I think the library version needs to be updated why the last changes that include the includeDeaccessioned flag as an argument of the use case